src/devices/acurite.c

```diff
@@ -471,8 +471,8 @@ static int acurite_txr_callback(bitbuffer_t *bitbuf) {
 		wind_speedmph = kmph2mph((float) wind_speed);
 		tempf = acurite_getTemp(bb[4], bb[5]);
 		tempc = fahrenheit2celsius(tempf);
-		humidity = acurite_getHumidity(bb[6]);
-
+		humidity = acurite_getHumidity(bb[6]);	
+		
```

Comment:
this white space change shouldn't be here.  The above indenting changes (possibly tabs to/from spaces) should also not be in this commit in my opinion.